index.js

```diff
@@ -58,30 +58,29 @@ const MusicControl = {
     }
   },
   on: function(actionName, cb){
-    if(subscription){
-      subscription.remove();
+    if(listenerOfNativeMusicControl == null){
```

Comment:
What about :
```js
if (!Boolean(listenerOfNativeMusicControl)){
``` 
or
```js
if (!listenerOfNativeMusicControl) {
```

Reply:
The second one is better, it also handles `undefined`

Reply:
the first as well ;-)